fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1706,6 +1706,12 @@ TEST_CASE("noSim/fdbserver/KeyValueStoreRocksDB/CheckpointRestore") {
 	return Void();
 }
 
+TEST_CASE("noSim/fdbserver/KeyValueStoreRocksDB/RocksDBTypes") {
+	ASSERT_EQ(sizeof(rocksdb::LiveFileMetaData), 184);
```

Comment:
Maybe add a comment on what to do if the assertion fails.

Reply:
Done.